bigquery/noxfile.py

```diff
@@ -40,14 +40,14 @@ def default(session):
         session.install('-e', local_dep)
 
     # Pyarrow does not support Python 3.7
-    if session.interpreter == 'python3.7':
+    if session.virtualenv.interpreter == '3.7':
```

Comment:
@theacodes What is the best way to get the python version in the new Nox API? This doesn't seem to work when `default` is run as its own session.